spyder_unittest/widgets/unittestgui.py

```diff
@@ -284,6 +234,8 @@ def set_running_state(self, state):
         ----------
         state : bool
             Set to True if tests are running.
+        stop_function : callable or None
+            Function to call when stop button is clicked
         """
         button = self.start_button
```

Comment:
Yes. I changed it to store the `TestRunner` instance. which has the added benefit that we keep a reference to it.